packages/amplify-graphql-searchable-transformer/src/__tests__/amplify-graphql-searchable-transformer.test.ts

```diff
@@ -299,7 +299,7 @@ test('it generates expected resources', () => {
             {
               Ref: Match.anyValue(),
             },
-            '"))\n$util.qr($ctx.stash.put("endpoint", "https://',
+            '"))\n$util.qr($ctx.stash.put("connectionAttributes", {}))\n$util.qr($ctx.stash.put("endpoint", "https://',
```

Comment:
Is this an expected change for `Searchable` resolver? Can you confirm whether we changed the corresponding querying logic to use 'connctionAttributes' variable?

Reply:
Please add an E2E for searchable.

Reply:
Confirmed that the search query does consume the `connectionAttributes`. The code block here is to have it initialized as it is applied to all `init` slot